consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/MutableQbftConfigOptions.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *  Copyright Hyperledger Besu Contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ *  the License. You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations under the License.
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.consensus.qbft;
+
+import org.hyperledger.besu.config.QbftConfigOptions;
+import org.hyperledger.besu.consensus.common.MutableBftConfigOptions;
+
+import java.util.Optional;
+
+public class MutableQbftConfigOptions extends MutableBftConfigOptions implements QbftConfigOptions {
```

Comment:
I like the structure and naming of the Mutable/Json ConfigOptions. It wasn't immediately clear to me how/why it was necessary.
I guess the need to mutate the config options is specific to transitions, so I wonder if a short sentence at the top of MutableQbftConfigOptions/JsonQbftConfigOptions would help shortcut a future reader's understanding?